deployment/apps/fc/radio_app/app_config.json

```
@@ -0,0 +1,90 @@
+        "deployment/system_definition/someip/servo_service/service.json",
+        "deployment/system_definition/someip/fc/recovery_service/service.json"
+    ],
+    "package": "srp.apps",
+    "adaptive_application": {
+        "RadioApp": {
+            "app": {
+                "functional_groups":[
+                    "Running",
+                    "SafetyMode"
+                ],
+                "parms": "",
+                "logger": {
+                    "app_id": "RAD-",
+                    "app_des": "czytanie i wysyłanie po radiu",
+                    "log_level": "kInfo",
+                    "log_mode": "kRemote",
+                    "ctx": [
+                        {
+                            "ctx_id": "ara",
+                            "log_level": "kInfo",
+                            "ctx_des": "Default ctx for ara"
+                        },
+                        {
+                            "ctx_id": "acom",
+                            "log_level": "kInfo",
+                            "ctx_des": "Default ctx for ara::com"
+                        },
+                        {
+                            "ctx_id": "adiag",
+                            "log_level": "kInfo",
+                            "ctx_des": "Default ctx for ara::diag"
+                        },
+                        {
+                            "ctx_id": "exec",
+                            "log_level": "kDebug",
+                            "ctx_des": "Default ctx for ara::exec"
+                        }
+                    ]
+                }
+            },
+            "provide": [
+                {
+                    "name": "RadioService as RadioService_ipc",
+                    "on": "ipc",
+                    "instance": 2
+                },
+                {
+                    "name": "RadioService as RadioService_udp",
+                    "on": "udp",
+                    "port": "10002",
+                    "instance": 1
+                }
+            ],
+            "require": [
+                {
+                    "name": "srp.env.EnvApp as EnvApp",
+                    "on": "ipc",
+                    "instance": 2
+                },
+                {
+                    "name": "srp.apps.GPSService as GPSService",
+                    "on": "ipc",
+                    "instance": 2
+                },
+                {
+                    "name": "srp.apps.PrimerService as PrimerService",
+                    "on": "ipc",
+                    "instance": 2
+                },
+                {
+                    "name": "srp.apps.ServoService as ServoService",
+                    "on": "ipc",
+                    "instance": 2
+                },
+                {
+                    "name": "srp.apps.RecoveryService as RecoveryService",
+                    "on": "ipc",
+                    "instance": 2
+                }
```

ipc(Inter-Process Communication) dziala na 1 komputerze a przypomne ze np Servo service jest na 2 komputerze
```suggestion
                {
                    "name": "srp.env.EnvApp as EnvApp",
                    "on": "udp",
                    "instance": 2
                },
                {
                    "name": "srp.apps.GPSService as GPSService",
                    "on": "ipc",
                    "instance": 2
                },
                {
                    "name": "srp.apps.PrimerService as PrimerService",
                    "on": "udp",
                    "instance": 2
                },
                {
                    "name": "srp.apps.ServoService as ServoService",
                    "on": "udp",
                    "instance": 2
                },
                {
                    "name": "srp.apps.RecoveryService as RecoveryService",
                    "on": "ipc",
                    "instance": 2
                }
```